### ISSUE TITLE
Brave Shields causes WebGL bugs

### ISSUE THREAD

**Issue by JoeyPacini** [brave/brave-browser#9618]
## Description 

Brave Shields causes WebGL bugs. If you turn it off, everything works fine.

## Steps to Reproduce

   1. Open in brave : https://codepen.io/cx20/pen/LbdEq (A simple test of Pixi.js with Canvas).
   2. See that it does not work.
   3. Disable Brave Shield or open it in another browser like Google Chrome or Firefox.
   4. See that it works.

## Actual result:

With Brave Shields ON : 

<a href="https://ibb.co/kGSV7Dg"><img src="https://i.ibb.co/9prmD49/fail.png" alt="fail" border="0"></a>

With Brave Shields OFF : 

<a href="https://ibb.co/dPgjfhN"><img src="https://i.ibb.co/fkvqXc3/good.png" alt="good" border="0"></a>

## Expected result:
Pixi WebGL working with Brave Shield ON

## Reproduces how often: 

Everytime

## Brave version (brave://version info)

Brave | 1.8.86 Chromium: 81.0.4044.129 (Official Build) (64-bit)
-- | --
Revision | 3d71af9f5704a40b85806f4d08925db24605ba25-refs/branch-heads/4044@{#979}
OS | Windows 10 OS Version 1809 (Build 17763.1158)
JavaScript | V8 8.1.307.31
User Agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.129 Safari/537.36

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes.
- Can you reproduce this issue with the beta channel? Yes.
- Can you reproduce this issue with the dev channel? Yes.
- Can you reproduce this issue with the nightly channel? Yes.

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? Yes.
- Does the issue resolve itself when disabling Brave Rewards? No. 
- Is the issue reproducible on the latest version of Chrome?  No.

## Miscellaneous Information:

With Brave Shields always ON : When I run Pixi.js with Node & Webpack Dev Server from localhost, it works fine but when I access it from my local IP (192.168.1.x) , it says "WebGL unsupported in this browser". Don't know how Brave Shields works but it's pretty strange.

With localhost : 

<a href="https://ibb.co/cbf8Xx3"><img src="https://i.ibb.co/hXQd1C9/works-2.png" alt="works-2" border="0"></a>

With Local IP : 

<a href="https://ibb.co/phQTmNJ"><img src="https://i.ibb.co/QDkV5hr/fail2.png" alt="fail2" border="0"></a>

Again, if I disable Brave Shields, it works fine with Local IP : 

<a href="https://ibb.co/MchNkYD"><img src="https://i.ibb.co/2v6K3D7/Good-3.png" alt="Good-3" border="0"></a>

As we can see, Brave Shields blocks something it sould not.

**Comment by rebron** [comment]
cc: @pes10k for prioritization

**Comment by pes10k** [comment]
@rebron its fixed in 1.11 and anywhere farbling has been uplifted.  The issue was the old, block WebGL in 3p-contexts defenses we had, and what farbling replaces.

Another webcompat farbling win :)

**Comment by kjozwiak** [comment]
Verification **PASSED** on `macOS 10.15.5 x64` using the following build:

```
Brave | 1.11.64 Chromium: 83.0.4103.97 (Official Build) dev (64-bit)
-- | --
Revision | 326d148b9655369b86498d9ecca39f63dd2bdd2d-refs/branch-heads/4103@{#657}
OS | macOS Version 10.15.5 (Build 19F101)
```

* Ensured that the STR/Case outlined via https://github.com/brave/brave-browser/issues/9618#issue-612213551 worked as expected
   * verified with both shields enabled and disabled

<img width="821" alt="Screen Shot 2020-06-15 at 5 01 22 PM" src="https://user-images.githubusercontent.com/2602313/84705803-f5c3ef00-af29-11ea-9b31-295e649dbd0e.png">
